src/gui/open.sh

```diff
@@ -0,0 +1,51 @@
+gui_open() {
+    local file output err msg_type target_file
+
+    file=$(yad \
+               --file \
+               --title="EasyGnuPG | Open file"\
+               --file-filter="Sealed files | *.sealed" \
+        ) || return 0
+
+    target_file="${file%.*}"
```

Comment:
Why not use `${file%.sealed}` (the same that is used in `cmd_open`)?

Reply:
No particular reason. It was what came to my mind first. BTW I have changed it to similar the cmd_open one now.